mvn_setup/tasks-parent/tasks-ui/src/core/TaskPriorities/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { SxProps, MenuList, MenuItem, ListItemText, Box, Button, Typography } from '@mui/material';
+import EmojiFlagsIcon from '@mui/icons-material/EmojiFlags';
+import { FormattedMessage } from 'react-intl';
+
+import Client from '@taskclient';
+import { usePopover } from 'core/TaskTable/CellPopover';
+
+const priorityColors = Client.PriorityPalette;
+const priorityOptions: Client.TaskPriority[] = ['LOW', 'HIGH', 'MEDIUM'];
+
+function getPrioritySx(priority: Client.TaskPriority): SxProps{
+  const color = priorityColors[priority];
+  return { color, ':hover': { color }, mr: 1 };
+}
+
+const TaskPriorities: React.FC<{ task: Client.TaskDescriptor, showPriorityText?: boolean }> = ({ task, showPriorityText }) => {
```

Comment:
The variable name `showPriorityText` sounds like a function name, but it's not a function -- it's just a boolean flag. This could be renamed like `priorityTextEnabled` or something like this.